### ISSUE TITLE
[A11Y] OJS/OPS/OMP Reader View -  site navigation missing "Home" link

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12655]
## Issue Description
There is no direct navigation link back to the main site, which hinders consistent navigation.

## How to reproduce / Current behavior
1. Go to OJS/OPS home
2. View the OJS/OPS top navigation.
3. Observe the lack of a "Home" or "Main Page" link in the menu.

## What application are you using?
OJS/OPS version 3.5.0

## Stack used 
- Device: Desktop
- OS: Windows/macOS
- Browser: Chrome browser/Firefox

## Testing method
- Method: Manual test

## Action
1. Add a explicit link to the "Main Page" in the navigation area.

### Screenshot

<img width="665" height="216" alt="OJS top navigation missing a Main site link menu item" src="https://github.com/user-attachments/assets/8168e8ee-9fb8-47e7-aa5c-be5c9074c18d" />

### WCAG Success Criteria
3.2.3 Consistent Navigation (AA)
